### ISSUE TITLE
Add alt text to image links 

### ISSUE THREAD

**Issue by emgrey02** [novelrt/novelrt.github.io#45]
I ran a lighthouse report and an accessibility concern is the lack of alt text on the two image links in the footer - the discord and github icons. 

I believe we just need to add a <title> tag to the svg elements.

**Comment by emgrey02** [comment]
May I be assigned this issue? :)

**Comment by RubyNova** [comment]
All yours. That said I do wonder if we don't need alt-text elements? It's been a long time since I've done HTML (about 6 years) so I could be wrong on that. :)
